postcss.config.js

```diff
@@ -0,0 +1,11 @@
+const postcss = require('postcss');
```

Comment:
I've given this some more thought and I think the defaults from the `index.js` could probably go. I get where you were coming from with doing as little as possible but right now it doesn't use enough plugins to compile the full SUIT library, which probably means that need to change. 

Rather than keeping defaults in the `index.js` and a `postcss.config.js` let's just keep the latter. I can't see people changing these options really otherwise they'd need to go into their global `suitcss` directory and install more postcss plugins.